app/plugin.go

```diff
@@ -182,6 +189,78 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 	a.SyncPluginsActiveState()
 }
 
+// SynchPlugins will synchronize the plugins installed locally
```

Comment:
nit: suggest present tense:
```suggestion
// SynchPlugins synchronizes the plugins installed locally
```